models/news-models.js

```diff
@@ -81,6 +81,14 @@ exports.addComment = (article_id, { body, author }) => {
 }
 
 exports.updateArticleVotes = (article_id, inc_votes) => {
+    if (inc_votes === undefined) {
+        return Promise.reject({ status: 400, msg: 'Missing required information' })
```

Comment:
psql will handle this error for you, meaning we can call the db without handling this first. psql will throw a 23502 error, so we can make use of psql error handler to send different messages according to different code